libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/camera/CameraComponent.kt

```diff
@@ -126,8 +128,10 @@ internal class CameraComponent constructor(
         }
     }
 
+    @OptIn(FlowPreview::class)
     private fun syncNavigationCameraState() {
-        navigationCamera.flowNavigationCameraState().observe {
+        // debounce is necessary to skip intermediate IDLE state when transitioning to FOLLOWING
+        navigationCamera.flowNavigationCameraState().debounce(10).observe {
```

Comment:
🤔  i know you say in the description debounce is not the preferred approach.

What will it take to fix it without a debounce? Because this will only make the issue happen less